bitnami/kubeapps/templates/frontend/configmap.yaml

```diff
@@ -22,11 +29,11 @@ data:
     # Google Kubernetes Engine requires the access_token as the Bearer when talking to the k8s api server.
     proxy_set_header Authorization "Bearer $http_x_forwarded_access_token";
     {{- end }}
-    {{- range .Values.clusters }}
+{{- range .Values.clusters }}
   {{- if .certificateAuthorityData }}
   {{ .name }}-ca.pem: {{ .certificateAuthorityData }}
   {{- end }}
-  {{- end}}
+{{- end}}
   vhost.conf: |-
```

Comment:
We have detected a small issue here, Michael sent this PR https://github.com/kubeapps/kubeapps/pull/2768 fixing it.
The point is that the `--pinniped-proxy` flag is not using the helper for getting the full name.

It'd be great to get it also solved here, no?

Reply:
I've created a PR to merge the change into this branch at https://github.com/juan131/charts/pull/4